app/controllers/coronavirus_form/able_to_leave_controller.rb

```diff
@@ -20,12 +20,19 @@ def submit
       render controller_path
     else
       update_session_store
+      write_responses
       redirect_to results_path
     end
   end
 
 private
 
+  def write_responses
+    time = Time.current
+    time_hour_floor = time - time.sec - time.min % 60 * 60
```

Comment:
I think this might be one of those rare occasions when a comment in the code could be helpful to explain why this line is important

Reply:
controversial! but agreed, have added this in

Reply:
Definitely.